src/trend/repository.md

```diff
@@ -28,7 +28,7 @@ structure Restock
 endstructure
 ```
 
-You'll notice that these structures all have the `ItemId` field in common. This is going to be how we keep track of which items are which. We'll use this field to join the data together later on. We'll also be using the `ItemId` to look up the item name in the `InventoryItem` structure. This is a common pattern that you'll see a lot of in the wild. It's a good idea to keep this in mind when you're designing your own data structures. Now lets take a look at the SDL for these structures.
+You'll notice that these structures all have the `ItemId` field in common. This is going to be how we keep track of which items are which. We'll use this field to join the data together later on. We'll also be using `ItemId` to look up the item name in the `InventoryItem` structure. This is a common pattern that you'll see frequently in the wild. It's a good idea to keep this in mind when you're designing your own data structures.<!--Keep what in mind?--> Now let's take a look at the SDL for these structures.
```

Comment:
The pattern to keep in mind is that a structure will have a common field like 'ItemId' that can be used to perform lookups across all of the related stuctures.